.circleci/config.yml

```diff
@@ -1,6 +1,34 @@
 version: 2
 jobs:
   build:
+    docker:
+      - image: circleci/python:3.5-stretch
+    steps:
+      - checkout
+
+      - run:
+          name: Install Debian packaging dependencies and download wheels
+          command: |
+            mkdir ~/packaging && cd ~/packaging
+            git clone https://github.com/freedomofpress/securedrop-debian-packaging.git
+            cd securedrop-debian-packaging
+            make install-deps && make fetch-wheels
+      - run:
+          name: Tag and make source tarball
+          command: |
+            cd ~/project
+            git tag 1000.0  # Dummy version number, doesn't matter what we put here
+            python3 setup.py sdist
```

Comment:
This will build a tarball the version mentioned in the `setup.py`, not with the git tag.